tests/integration/ignore_root_user_error/WORKSPACE

```diff
@@ -1,3 +1,19 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "protobuf",
+    sha256 = "da288bf1daa6c04d03a9051781caa52aceb9163586bff9aa6cfb12f69b9395aa",
+    strip_prefix = "protobuf-27.0",
+    url = "https://github.com/protocolbuffers/protobuf/releases/download/v27.0/protobuf-27.0.tar.gz",
+)
+
+http_archive(
+    name = "rules_cc",
+    sha256 = "d9bdd3ec66b6871456ec9c965809f43a0901e692d754885e89293807762d3d80",
+    strip_prefix = "rules_cc-0.0.13",
+    urls = ["https://github.com/bazelbuild/rules_cc/releases/download/0.0.13/rules_cc-0.0.13.tar.gz"],
+)
```

Comment:
@rickeylev, I think this means that the `py_repositories` function adding these deps needs to be updated. This looks like the wrong thing to do if we want to keep supporting `bazel@HEAD` with a released `rules_python` version?

Reply:
Good point. Something felt off updating so many workspace files for tests. I've started https://github.com/bazelbuild/rules_python/pull/2335 to update py_repositories.